bin/setup

```diff
@@ -5,7 +5,7 @@ if ! [ -x "$(command -v node)" ]; then
   echo 'Error: node is not installed.' >&2
 
   if ! [ -x "$(command -v brew)" ]; then
-    echo 'Go here to install: https://nodejs.org/en/download/'
```

Comment:
This was actually intended to manually download node if brew is not here I think